heron/common/src/python/network/heron_client.py

```diff
@@ -157,7 +162,7 @@ def handle_write(self):
     self.total_pkt_written += num_pkt_written
 
   def writable(self):
-    if self.connecting:
```

Comment:
Travis machine uses Python 2.7.3, in which asyncore module doesn't have `connecting` attribute.